addon/index.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 
 Ember.deprecate("ember-getowner-polyfill is now a true polyfill. Use Ember.getOwner directly instead of importing from ember-getowner-polyfill", false, {
-  id: "ember-getowner-polyfill.import"
+  id: "ember-getowner-polyfill.import",
+  until: '3.0.0'
```

Comment:
Yeah, it's confusing. I'm not sure we even have any tooling that can distinguish between deprecations in ember vs addons, and make sense of their different version numbers.